packages/microcosm/src/action.js

```diff
@@ -54,6 +55,7 @@ class Action extends Emitter {
     this.children = []
     this.revisions = []
     this.origin = origin
+    this.connected = true
```

Comment:
I didn't like how we determined if an action was disconnected or not based on the presence of `next` or `parent`.